CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # CHANGELOG
+## 0.0.37
+- [commands] Improve SubCommand serialization with Command Lexer
+- [commands] Add Parameter names for argument parsers
```

Comment:
didn't you say this will be in another pr?

Reply:
opss

Reply:
but it's true